File: swagger/models/get_packages_response.py
from .base_model import BaseModel

from .package import Package
from .pagination_response import PaginationResponse

class GetPackagesResponse(BaseModel):
	"""
	no description available

	Attributes:

		packages
		Contains information about the resulting packages.

		pagination_response
		Contains information about the pagination used.
	"""

	input_map = {
		'Packages': ['packages', Package],
		'PaginationResponse': ('pagination_response', PaginationResponse),
		}

	output_map = {
		'packages': 'Packages',
		'pagination_response': 'PaginationResponse',
		}

	packages = None
	pagination_response = None

File: swagger/Models/GetPackagesResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetPackagesResponse extends BaseModel
{
	/**
	 * Contains information about the resulting packages.
	 * 
	 * @var Package[] 
	 */
	public $packages;
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;

	protected function getInputMap()
	{
		return [
			'Packages' => ['packages', 'array', Package::class],
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'packages' => ['Packages', 'array', Package::class],
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			];
	}
}
